content/_index.html

```diff
@@ -55,7 +55,7 @@ <h2>Do What I Mean. Good Docs. Clean Code. Sounds good?</h2>
         provide support for rotated monitors.
     </li>
     <li>
-        Use a tree as data structure. This allows for more flexible layouts than
+        Use a tree as a data structure. This allows for more flexible layouts than
         the column-based approach used by other window managers.
```

Comment:
```suggestion
        Use a tree as the data structure. This allows for more flexible layouts
        than the column-based approach used by other window managers.
```

Reply:
Yeah, I considered that, but it somehow also feels clunky to me. Maybe that’s just me and it’s the better choice than what we have now 🤷